app2/src/main/java/com/example/recipebox/ui/theme/Theme.kt

```diff
@@ -7,15 +7,16 @@ import androidx.compose.material.lightColors
 import androidx.compose.runtime.Composable
 
 private val DarkColorPalette = darkColors(
-    primary = Purple200,
-    primaryVariant = Purple700,
-    secondary = Teal200
+    primary = Red200,
+    primaryVariant = Red700,
+    secondary = Blue200
 )
 
 private val LightColorPalette = lightColors(
-    primary = Purple500,
-    primaryVariant = Purple700,
-    secondary = Teal200
```

Comment:
Changed the variable name of the primary, secondary, etc to match what the color should represent, aka the color in your design doc